core/block_validator.go

```diff
@@ -247,6 +248,26 @@ func ValidateHeader(config *ChainConfig, pow pow.PoW, header *types.Header, pare
 			return &BlockNonceErr{header.Number, header.Hash(), header.Nonce.Uint64()}
 		}
 	}
+	// DAO hard-fork extension to the header validity: a) if the node is no-fork,
```

Comment:
Is there a need to document this formally, so that other node-implementations can comply?